### ISSUE TITLE
Release G2G pose estimation models on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [WeiYuFei0217/G2G#1]
Hi @WeiYuFei0217 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work on Arxiv and was wondering whether you would like to submit it to [hf.co/papers](https://hf.co/papers) to improve its discoverability. If you are one of the authors, you can submit it at https://huggingface.co/papers/submit.

The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance), and you can also claim the paper as yours which will show up on your public profile at HF.

Would you like to host the 10 pre-trained G2G weights you've released on https://huggingface.co/models? I see you're currently using Baidu Cloud for storage. Hosting on Hugging Face will give you significantly more visibility and enable better discoverability for the global research community. 

I also noticed you mentioned that the current `facebook/map-anything` weights on HF are incompatible with your G2G modules. We could also host the specific version of the backbone you used as part of your model organization on HF. This would make it incredibly easy for others to reproduce your results with a simple `from_pretrained` call!

If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). If it's a custom PyTorch model, you can use the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to the model, or you can directly upload through the UI and use [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file).

After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your work.

Let me know if you're interested/need any guidance :)

Kind regards,

Niels

**Comment by WeiYuFei0217** [comment]
Hi Niels,
thanks so much for reaching out, this is really helpful! 🤗

We've now uploaded everything to the Hub: https://huggingface.co/feixue22/G2G
It includes:
- the 10 pretrained G2G weights (release_weights/)
- the example sanity-check bundles and the paper-subset evaluation results
- the exact compatible MapAnything backbone (DINOv2-large / 1024-dim, v1.0.1), since the current facebook/map-anything weights are the newer giant/1536 variant that isn't compatible with our modules

Code, installation and full usage live on our GitHub repo (https://github.com/WeiYuFei0217/G2G), which the model card links to. The model also already shows up under the paper page (https://huggingface.co/papers/2606.08284).

Thanks again!

Best,
Yufei

**Comment by NielsRogge** [comment]
Hey @WeiYuFei0217, thanks so much for uploading the models to Hugging Face and linking them in the README. This is great for the community.

If you are one of the authors of the paper, you can also claim the paper page by clicking on your name at https://huggingface.co/papers/2606.08284, which will show it on your Hugging Face profile.

Thanks again for making your work accessible on the Hub!
